src/helix/core.cljs

```diff
@@ -217,6 +217,39 @@
 
   (rds/renderToString ($$ MyComponent {:foo "baz"})))
 
+(defn ref
```

Comment:
thinking more, perhaps mirroring React's API is better for finding docs - and it is longer than `use-ref` so people are more likely to use the right API 😄 

```suggestion
(defn create-ref
```